layers/+lang/ipython-notebook/packages.el

```diff
@@ -65,116 +84,20 @@
                        'warn (format "ipython-notebook/init-ein: undefined %s"
                                      (cl-second bind))))))
                 (copy-tree bindings)))
-	(eval (append '(spacemacs|define-transient-state
```

Comment:
I see now why you needed "tkamat"'s bindings.  It's because spacemacs has some clever check to ensure the hardcoded docstring is "mated" to the bindings.  I am sympathetic to the hardcoded docstring, but as the primary EIN developer, the bindings from 72ea049 are highly advisable, if not non-negotiable, especially as the `-km` is important.

Reply:
Yes but even then it does work for other keybindings too